pkg/varnishcluster/controller/varnishcluster_grafana_dashboard_template.go

```diff
@@ -2342,20 +2341,26 @@ var grafanaDashboardTemplate =
       {
         "allValue": null,
         "current": {
+          "selected": false,
           "text": "All",
           "value": "$__all"
         },
-        "datasource": "{{.DatasourceName}}",
```

Comment:
Here you change the `.DatasourceName` to `Prometheus-varnish`, but you don't remove the field from the `data` map. Did you mean to do this?

Reply:
No, I accidentally changed that. Reverted. Good catch.

Reply:
Indeed! good catch